### PR TITLE
Development flist volumes info main net

### PR DESCRIPTION
**Pull request by RafyAmgadBenjamin**
### Description

Add volume description in a `generic container` solution

### Related Issues

https://github.com/threefoldtech/js-sdk/issues/1347

### CODE REVIEW

**Review by waleedhammam — APPROVED** [review]
LGTM

**Review by dmahmouali — APPROVED** [review]
Should we port this for apps that have volumes?

File: jumpscale/packages/admin/frontend/components/solutions/Info.vue [hunk 1]
```diff
@@ -36,7 +36,7 @@ module.exports = {
         cancelSolution: false,
       },
       KeysWithTypeList: ["Node ids", "wids", "Active workload ids"],
-      KeysWithTypeDict: ["nodes"]
+      KeysWithTypeDict: ["nodes", "Volumes"]
```
Comment (xmonader) [inline jumpscale/packages/admin/frontend/components/solutions/Info.vue:39]: Should it be Capitalized or lowercase like nods

Comment (RafyAmgadBenjamin) [reply jumpscale/packages/admin/frontend/components/solutions/Info.vue:39]: the key returned as it is from the backend, most the keys are capitalized except nodes and we are not sure where it is used.

**Pull request merged**